cliqz/index.js

```diff
@@ -19,11 +21,20 @@ class BrowserCoreApp extends React.Component {
     this.state = { results: [], cliqz: null };
   }
 
+  onAction = ({ module, action, args, id }) => {
+    return this.loadingPromise.then(() => {
+      return this.state.cliqz.app.modules[module].action(action, ...args).then((response) => {
+        if (typeof id !== 'undefined') {
+          NativeModules.Bridge.replyToAction(id, response);
+        }
+        return response;
+      });
+    });
+  }
+
   componentWillMount() {
     const app = new App();
```

Comment:
Maybe you want to initialize in the constructor and use `this.app` here?

Reply:
does not matter really